verifiers/plonky2/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "pallet-plonky2-verifier"
+version = "0.1.0"
+description = "A plonky2 verifier pallet implementation based on pallet-verifiers abstraction."
+homepage.workspace = true
+edition.workspace = true
+authors.workspace = true
+repository.workspace = true
+license = "Apache-2.0"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+codec = { workspace = true }
+scale-info = { workspace = true }
+sp-core = { workspace = true }
+hp-verifiers = { workspace = true }
+log = "0.4.20"
+hex-literal = { workspace = true, optional = true }
+frame-system = { workspace = true }
+frame-benchmarking = { workspace = true, optional = true }
+pallet-verifiers = { workspace = true }
+sp-io = { workspace = true, optional = true }
+frame-support = { workspace = true }
+plonky2-verifier = {git = "https://github.com/distributed-lab/plonky2-verifier", default-features = false}
```

Comment:
Can we create a tag and we reference it ? To avoid unwanted updates

Reply:
Sure we can

Reply:
Set this to `0.1.0`